packages/ember-data/lib/serializers/json-serializer.js

```diff
@@ -1224,6 +1224,22 @@ export default Serializer.extend({
     return this.extractArray(store, typeClass, payload, id, requestType);
   },
   /**
+    `extractQueryRecord` is a hook into the extract method used when a
+    call is made to `DS.Store#queryRecord`. By default this method is an
+    alias for [extractSingle](#method_extractSingle).
+
+    @method extractQueryRecord
+    @param {DS.Store} store
+    @param {DS.Model} typeClass
+    @param {Object} payload
+    @param {(String|Number)} id
+    @param {String} requestType
+    @return {Object} object A hash of deserialized object
+  */
+  extractQueryRecord: function(store, typeClass, payload, id, requestType) {
```

Comment:
I think its ok to leave this in to help smooth the transition to the new style hooks. @wecc is a `normalizeQueryRecordResponse` method needed?